src/components/Header.js

```diff
@@ -3,7 +3,7 @@ import Nav from './Nav';
 
 function Header (props) {
   return(<header>
-    <h1>Vibe Check</h1>
+    <h2>Vibe Check</h2>
```

Comment:
Is the change of header level for sizing? If so, I think the size could be fixed in CSS, but the header level is useful for page hierarchy and accessibility. For instance a h2 can't appear if a h1 hasn't yet.